main.go

```diff
@@ -5,10 +5,10 @@ import (
 	"fmt"
 	"net/http"
 	"path/filepath"
-	"strconv"
 )
 
 var (
+	ip   = flag.String("ip", "", "Address to bind to")
```

Comment:
Generally, both host:port should be one flag called listen address or just `listen` in flag. With that it can accept both format, ":8080" or "host:8080". I know it's been wrong (adding port arg) before this PR, but better if we can fix it along with this PR.

Tip by Brad in this video https://youtu.be/1rZ-JorHJEY?t=16m11s.